doc/appendices/administration.rst

```diff
@@ -82,6 +82,69 @@ GitHub is the official record of roles.
 
 .. _team list: https://github.com/orgs/celeritas-project/teams
 
+
+.. _code_review:
+
+Code review
+===========
+
+Reviewing incoming code maintains (and should improve) the quality of the
+codebase in the interests of correctness and long-term maintainability.
+If the code is reviewed according to the guidelines below, at least two people
+(the author and the reviewer) will be able to modify any given piece of code,
+increasing the `bus factor`_.
+
+.. _bus factor: https://en.wikipedia.org/wiki/Bus_factor
+
+Review process
+--------------
+
+Each pull request must be reviewed by at least one
+member of the :ref:`core team <roles>` who is knowledgeable about
+the section of code being modified.
+
+The review process must be based on
+*constructive feedback* ("here's a suggestion to make this better" or "did you
+consider what would happen if X?"), not *destructive feedback* ("this code is
+ugly"). When reviewing, you should focus almost exclusively on the new
+changeset, as opposed to larger systemic issues, or smaller problems with
```

Comment:
> as opposed to larger systemic issues, 

Do we mentioned somewhere what the reviewer should do if while review a PR, they notice an unrelated or systemic issue?  (Without this mention the implication could then seem to be: just ignore them :( ).

Reply:
Hehe, good point. Maybe open up a new issue or fix with a pull request? Or if it dovetails with the current one, mention it and if the submitter wants to add some new changes, cool, but it shouldn't be a requirement?

Reply:
So maybe "It is at the discretion of the PR author further those kind of unrelated issues are addressed within the current PR or if the reviewer should open an issue to ensure that the issues are addressed at some point (or better yet solved immediately through a new PR)"